### DISCUSSION TITLE
Issue with Github application on SteamOS

### DISCUSSION THREAD

**Discussion by axzr** [community/community#48817] [Other Feature Feedback, Questions, & Ideas]
I'm trying to get Git access set up on my Steam Deck and I'm having some issues.

The short version is that I've installed Github from the Discover app but when I run it I get the error 'Can't find session /org/freedesktop/secrets/session/2'

I've tried googling around but haven't found a good solution yet.

This seems like a pretty basic thing? I'm sure I'm doing something simple wrong?

**Comment by sbhask** [comment]
https://murmele.github.io/Gittyup/

this might work

**Reply by axzr** [reply]
Thanks for the suggestion Sam. I'd very much like to get the GIthub app working though.
